bitcoin.js

```diff
@@ -44,9 +44,9 @@ function decodepsbt({ psbt }) {
     return resp
 }
 
-function listtransactions() {
+function listtransactions({ label = '"*"', count }) {
     check_wallet()
-    const resp = JSON.parse(child_process.execSync(`${bitcoin_command} -rpcwallet=${process.env.BITCOIN_WALLET} listtransactions`))
+    const resp = JSON.parse(child_process.execSync(`${bitcoin_command} -rpcwallet=${process.env.BITCOIN_WALLET} listtransactions ${label} ${count ? count : '10'}`))
     return resp
```

Comment:
```count ? count : '10'```
any specific reason of not using default argument like label, `{ label = '"*"', count = '10'}?
because if caller passes `0` which is valid number, but ternary will end-up returning '10' as 0 is falsy.

Reply:
yeah it makes no sense to call the function with count = 0, but good thinking

Reply:
ill update the logic to handle though

Reply:
ah yes much cleaner with the default var set - thanks